grammar.js

```diff
@@ -334,12 +336,13 @@ module.exports = grammar({
 
       const hex_literal = seq(
         choice('0x', '0X'),
+		    optional('.'),
```

Comment:
I don't think lua accepts `.` after `0x` or `0X`.

Reply:
Nevermind looks like Luajit supports it.

Reply:
https://github.com/lua/lua/blob/e15f1f2bb7a38a3c94519294d031e48508d65006/testes/math.lua#L507